docs/Quickstart.mdx

```diff
@@ -0,0 +1,112 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Quick Installation
+
+This is the quick installation guide for Enarx. The goal is to help you get started using Enarx right away for development/debugging purposes on Windows, macOS, and Linux.
+
+:::note
+To build from source, please refer to the [complete installation guide](https://enarx.dev/docs/Install).
+:::
+
+:::note
+Full [TEE](https://enarx.dev/docs/Start/TEE) (Intel SGX/AMD SEV-SNP) and KVM support is available with the published statically compiled Linux binary.
+:::
+
+<Tabs>
+  <TabItem value="windows" label="Windows">
```

Comment:
Since the first tab is what is shown by default, I think it's important that we make the first tab be the most common OS that developers will be using, which I think will probably be Linux or Mac. Do you agree?

Reply:
This is something Paul discussed with me. While Linux rules for deployment, for development it's not the case. This is the best data I could find for professional developers: 41% Windows, 30% macOS, 25% Linux
https://insights.stackoverflow.com/survey/2021#most-popular-technologies-op-sys